Action.swift

```diff
@@ -23,83 +23,75 @@ public final class Action<Input, Element> {
     /// Thus, inputs count equals elements count + errors count.
     public let inputs = PublishSubject<Input>()
-    private let _completed = PublishSubject<Void>()
+    fileprivate let _completed = PublishSubject<Void>()
 
     /// Errors aggrevated from invocations of execute(). 
     /// Delivered on whatever scheduler they were sent from.
     public var errors: Observable<ActionError> {
         return self._errors.asObservable()
     }
-    private let _errors = PublishSubject<ActionError>()
+    fileprivate let _errors = PublishSubject<ActionError>()
 
     /// Whether or not we're currently executing. 
     /// Delivered on whatever scheduler they were sent from.
     public var elements: Observable<Element> {
         return self._elements.asObservable()
     }
-    private let _elements = PublishSubject<Element>()
+    fileprivate let _elements = PublishSubject<Element>()
 
     /// Whether or not we're currently executing. 
     /// Always observed on MainScheduler.
     public var executing: Observable<Bool> {
         return self._executing.asObservable().observeOn(MainScheduler.instance)
     }
-    private let _executing = Variable(false)
+    fileprivate let _executing = Variable(false)
     
     /// Observables returned by the workFactory.
     /// Useful for sending results back from work being completed
     /// e.g. response from a network call.
     public var executionObservables: Observable<Observable<Element>> {
         return self._executionObservables.asObservable().observeOn(MainScheduler.instance)
     }
-    private let _executionObservables = PublishSubject<Observable<Element>>()
+    fileprivate let _executionObservables = PublishSubject<Observable<Element>>()
     
     /// Whether or not we're enabled. Note that this is a *computed* sequence
     /// property based on enabledIf initializer and if we're currently executing.
     /// Always observed on MainScheduler.
     public var enabled: Observable<Bool> {
         return _enabled.asObservable().observeOn(MainScheduler.instance)
     }
-    public private(set) var _enabled = BehaviorSubject(value: true)
+    public fileprivate(set) var _enabled = BehaviorSubject(value: true)
 
-    private let executingQueue = dispatch_queue_create("com.ashfurrow.Action.executingQueue", DISPATCH_QUEUE_SERIAL)
-    private let disposeBag = DisposeBag()
+    fileprivate let executingQueue = DispatchQueue(label: "com.ashfurrow.Action.executingQueue", attributes: [])
+    fileprivate let disposeBag = DisposeBag()
 
-    public init<B: BooleanType>(enabledIf: Observable<B>, workFactory: WorkFactory) {
-        self._enabledIf = enabledIf.map { booleanType in
-            return booleanType.boolValue
+    public init<B: ExpressibleByBooleanLiteral>(enabledIf: Observable<B> = Observable.just(true), workFactory: @escaping WorkFactory) {
+        self._enabledIf = enabledIf.map { (booleanLiteral) -> Bool in
+            return booleanLiteral as! Bool
```

Comment:
[`BooleanType` has been renamed to `Bool`](https://github.com/apple/swift/blob/4a24b069e0388cdac02551cda06a0d0ef1298436/test/stdlib/Renames.swift#L124).
We can make this initializer non-generic and replace the type parameter with `Bool`.